testCourse/questions/addNumbers/info.json

```diff
@@ -3,5 +3,10 @@
     "title": "Add two numbers",
     "topic": "Algebra",
     "tags": ["mwest", "fa17", "tpl101", "v3"],
-    "type": "v3"
+    "type": "v3",
+    "sharingSets": [
+        "final-exam"
+    ],
+    "publiclyShared": true,
+    "publiclySharedWithSource": true
```

Comment:
Nit: `sharedPublicly` / `sharedPubliclyWithSource`? This would match the DB columns.

Reply:
done, thanks